### DISCUSSION TITLE
issue updating signalk

### DISCUSSION THREAD

**Discussion by shlandrews** [bareboat-necessities/lysmarine_gen#424] [Q&A]
I seem to be unable to update the signalk server to latest version through the signalk app -> Server -> Update. It starts then says 'Something went wrong' and need to restart signalk.
Current version 2.7.2 upgrading to 2.8.2
Have rebooted the device and checked internet connection etc. Not desperate to update it but there may be some reasons to in the future.

For reference the other device running OpenPlotter V2 has updated fine just now in the same manner of updating within the signalk app so I'm not sure it's signalk issue.

Thanks

**Comment by mgrouch** [comment]
SignalK update is not supported. In fact it is disabled in UI in latest version of BBN OS.

**Reply by shlandrews** [reply]
That's unfortunate, hope there are no useful features signalk release in the future. Is this purely related to the pre- installed plugins and if I don't worry about them and manage them myself, I can simply manually upgrade singalk from its repository or is it deeper than this and do not even recommend this?

**Reply by mgrouch** [reply]
Victron Venus OS does same. It’s better for stability of the system not to allow general users to update SignalK as it might lead to non working system. Advanced users can figure out from the source of BBN OS how to upgrade SignalK from command line.

**Reply by shlandrews** [reply]
Thanks for clarification, will keep an eye on it if anything particularly useful from signalk to warrant upgrade manually and test out, understand you can only do so much as to test what was available and stable at time of your release. Works fine for everything I need at the moment, so as we say in Australia, if it ain't broke don't fix it..

**Reply by Tosca36** [reply]
I know this is 2 years old but does the latest version of bbn allow signalk updates? I had to use command line and chat gpt to update. Seems ok for now, I am also using a marine control server from gedad so not sure if it will work.

**Reply by shlandrews** [reply]
At least from my experience when I upgraded signalk I started to get stability issues over time and so eventually reverted back to a clean install with the old 2.13 version. It is getting to a point now where some of my regularly used plugins seem to be no longer compatible with signalk v2.13 if update the plugins but not the signalk version

**Reply by Tosca36** [reply]
I was having issues with opencpn locking up so I decided to start from scratch with newest bbn OS. I could not create an account for signalk so I thought a signalk  update would fix it and it did, but if I'm going to have issues with signalk then I guess I need to start over again?

**Reply by shlandrews** [reply]
Was just sharing my experience, I'm not the developer so can't answer. For me a clean install and just never updating signalk plugins keeps things stable. If need all latest signalk and plugins I have another server running openplotter I use in parallel but would be nice to upgrade signalk on my BBN instance as well

**Comment by Tosca36** [comment]
Thank you for sharing!Sent from my iPhoneOn Sep 10, 2026, at 04:10, shlandrews ***@***.***> wrote:﻿
Was just sharing my experience, I'm not the developer so can't answer. For me a clean install and just never updating signalk plugins keeps things stable. If need all latest signalk and plugins I have another server running openplotter I use in parallel but would be nice to upgrade signalk on my BBN instance as well
